sound/soc/intel/skylake/skl.c

```diff
@@ -359,7 +359,7 @@ static int skl_resume(struct device *dev)
 	struct pci_dev *pci = to_pci_dev(dev);
 	struct hdac_bus *bus = pci_get_drvdata(pci);
 	struct skl_dev *skl  = bus_to_skl(bus);
-	struct hdac_ext_link *hlink = NULL;
```

Comment:
@lyakh can we drop the "(cosmetic)" in the commit title?

Reply:
@ranj063 sorry, don't understand, can you explain why in this specific patch? Or in all patches? I think it's perfectly appropriate, since these patches don't fix any bugs or checkpatch or compiler warnings, nor do they implement any features. They are cosmetic.

Reply:
@lyakh sorry just realized that this is there in all patches. Absolutely agree this is cosmetic and I thought its quite obvious, hence the suggestion. But up to you if you want to keep it.